index.html

```diff
@@ -5,11 +5,76 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel="stylesheet" href="./css/style.css">
     <title>Welcome to this Restaurant</title>
 </head>
 
 <body>
-
+    <header>
+    </header>
+    <main>
+        <section id="welcome" class="vh100 container d-flex flex-column justify-content-center">
+            <h1 class="position-relative">
+                <span>Restaurant - Bar - Lounge</span><br>
+                WELCOME TO BLABLA RESTAURANT
+                <img src=" ./images/PngItem_303341.png" alt="background image">
+                <img src="./images/wine-glass-red-wine-wine-cocktail-champagne-drink-wine-d3d72936183c1a59da94eb28e13d3e17.png"
+                    alt="background image 2">
+            </h1>
+        </section>
+        <section id="menu" class="container-fluid px-4">
+            <div class="h2-wrapper">
+                <h2>Menu
+                </h2>
+                <div class="line"></div>
+            </div>
```

Comment:
- [ ] **Required:** This page contains so many `div` tags which aren't the best for HTML5 semantic websites. Kindly replace with more[ semantic tags](https://www.freecodecamp.org/news/semantic-html-alternatives-to-using-divs/) such are `article`, etc.

Reply:
@brytebee as far as I know `article`'s are not used to reduce the number of `div`'s, but to point to the independently distributable content within a page as you can see [here](https://www.w3schools.com/tags/tag_article.asp)
I dont see how its semantically correct to use `article` tags here.

![article](https://user-images.githubusercontent.com/83017311/152153233-05e7de22-eeb4-48c0-8a5e-49a27e15c4c5.jpg)
.

Reply:
I see.